test/extended/syncer/syncer.go

```diff
@@ -0,0 +1,76 @@
+package syncer
+
+import (
+	"os"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area/transparent-multi-cluster]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-syncer")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke][BYO] Validate create, modifying, deleting a deployment from KCP gets synced to the pcluster", func() {
+		pclusterKubeconfig := os.Getenv("PCLUSTER_KUBECONFIG")
+		if pclusterKubeconfig == "" {
+			g.Skip("No pcluster kubeconfig set for the test scenario")
+		}
+		k.SetupWorkSpaceWithNamespace()
+		myWs := k.WorkSpace()
+		k.SetGuestKubeconf(pclusterKubeconfig)
+
+		g.By("# Create workload sync and generate syncer resources manifests in current workspace")
+		mySyncer := NewSyncTarget()
+		mySyncer.OutputFilePath = "/tmp/" + myWs.Name + "." + mySyncer.Name + ".yaml"
+		mySyncer.Create(k)
+		defer mySyncer.Clean(k)
+
+		g.By("# Apply syncer resources on pcluster and wait for synctarget become ready")
+		defer k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("delete").Args("-f", mySyncer.OutputFilePath).Execute()
+		err := k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("apply").Args("-f", mySyncer.OutputFilePath).Execute()
+		o.Expect(err).ShouldNot(o.HaveOccurred())
+		mySyncer.WaitUntilReady(k)
+		mySyncer.CheckDisplayAttributes(k)
+
+		g.By("# Create workload using the shared compute should work well")
```

Comment:
```suggestion
		g.By("# Create workload using the BYO compute should work well")
```

Reply:
Yeah, addressed.  BTW, what's the full name of BYO? Seems O means Openshift. In addition, the case should support many kinds of kubernetes clusters not only openshift.

Reply:
BYO means "Bring your own cluster" it could be any cluster that can be integrated with kcp

Reply:
Thanks for clarifying!